### ISSUE TITLE
Eweka: Error 430

### ISSUE THREAD

**Issue by muis** [spotnet/spotnet#19]
Eweka states on their support pages that the 430 errors are caused by Spotnet generating Message-ID's that are not always unique. There are mulitple reason why I doubt this is really the case, for example:
- Other providers with the same retention (Astraweb, Giganews) do not suffer from this issue.
- The current length of the Message-ID's is sufficient enough to make the chance of doubles highly unlikely.
- When such a message is requested, why does their server respond with 430 (does not exist) when in fact it exists twice?

And most of all: Message-ID's that are not unique should have been rejected by the server upon posting them. If you could really delete existing messages just by posting another one with the same Message-ID, that's good news for the content industry: they can censor any post they want :)

For the real cause: I guess that their servers require a GROUP command before every article request. The reason why Spotnet doesn't send this is that it prefers requesting articles by their number, and servers are supposed to remember the currently selected group when requesting articles this way. If retrieval by number fails, Spotnet retries again by using the Message-ID, but still doesn't send a GROUP command, causing the infamous 430 error on Eweka servers (because you're on the wrong spool-server).

To sum it up: Eweka servers seem to have a problem serving articles by their article-number sometimes, and the alternate method (by Message-ID) also fails because it requires a re-send of the GROUP command. This last issue can be fixed in a new release, but only they can fix the first.
